guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md

```diff
@@ -185,9 +185,9 @@ uniteUnique([1, 3, 2], [5, 2, 1, 4], [2, 1]);
 
 ### Code Explanation:
 
-*   We first use `concat()` with an empty array `<a href='http://exploringjs.com/es6/ch_maps-sets.html#_set' target='_blank' rel='nofollow'>]` as a starting point and the spread operator `...` to create an array out of the Arguments object and to flatten it at the same time
+*   We first use `concat()` with an empty array as a starting point and the spread operator `...` to create an array out of the Arguments object and to flatten it at the same time
 *   then we use the new ES2015 **Set** object to store only unique values
-*   (to learn more about Sets, read [here</a>)
+*   (To learn more about **Set**, [click here](http://exploringjs.com/es6/ch_maps-sets.html#sec_set)).
```

Comment:
As the Relevant Links section already has a link to about `Sets`, I suggest just to remove line 190 all together.  Everything else looks fine.

Reply:
Thanks for the feedback. I removed the line 190.